product/models.py

```diff
@@ -0,0 +1,136 @@
+    class Meta:
+        db_table            = 'colors'
+
+class Promotion(models.Model):
+    promotion         = models.CharField(max_length = 50, null = True)
+    
+    def __str__(self):
+        return self.promotion 
+    
+    class Meta:
+        db_table       = 'promotions'
+
+class Silouette(models.Model):
+    silouette          = models.CharField(max_length = 50)
+    
+    def __str__(self):
+        return self.silouette 
+    
+    class Meta:
+        db_table        = 'silouettes'
+
+class Size(models.Model):
+    size                = models.IntegerField(default = 0)
+
+    def __str__(self):
+        return self.size 
+    
+    class Meta:
+        db_table        = 'sizes'
+
+class Product(models.Model):
+    serial_number           = models.CharField(max_length = 50)
+    category                = models.ForeignKey(Category, on_delete = models.CASCADE)
+    sub_category            = models.ForeignKey(SubCategory, on_delete = models.CASCADE)
+    serise                  = models.ForeignKey(Series, on_delete = models.CASCADE)
+    main_image              = models.URLField()
+    hover_image             = models.URLField()
+    price                   = models.IntegerField(default = 0)
+    discount_rate           = models.IntegerField(default = 0)
+    sex                     = models.ForeignKey(Sex, on_delete = models.CASCADE)
+    simple_description      = models.CharField(max_length = 200)
+    color                   = models.ForeignKey(Color, on_delete = models.CASCADE)
+    size                    = models.ManyToManyField(Size) 
+    promotion               = models.ForeignKey(Promotion, on_delete = models.CASCADE)
+    silouette               = models.ForeignKey(Silouette, on_delete = models.CASCADE)
+    score                   = models.DecimalField(max_digits = 5, decimal_places = 2)
+    created_at              = models.DateTimeField(auto_now_add = True)
+    updated_at              = models.DateTimeField(auto_now = True) 
+
+    def __str__(self):
+        return self.serial_number 
+    
+    class Meta:
+        db_table            = 'products'
+
+class Medium(models.Model):
+    product                 = models.ForeignKey(Product, on_delete = models.CASCADE)
+    medium_url              = models.URLField()
+    medium_type             = models.CharField(max_length = 100)
+    
+    class Meta:
+        db_table            = 'media'
+
+class ProductInformation(models.Model):
+    product                 = models.ForeignKey(Product, on_delete = models.CASCADE)
+    material                = models.CharField(max_length = 100) 
+    minimum_size            = models.IntegerField(default = 0)
+    maximum_size            = models.IntegerField(default = 0)
+    manufacture             = models.CharField(max_length = 100)
+    country                 = models.CharField(max_length = 100)
+    caution                 = models.CharField(max_length = 200)
+    quality_assureance      = models.CharField(max_length = 100)
```

Comment:
```suggestion
    quality_assurance      = models.CharField(max_length = 100)
```
오타인가요?
이런거 꼼꼼히 보셔야 합니다. 왜냐하면 모델이 한 번 생성되고나면, 나중에 데이터베이스의 속성이나 설정을 바꾼다는 것은 규모가 큰 작업이고, 비용 지출도 큽니다. 때문에 데이터베이스는 처음 구축할 때 조심하셔야 합니다.